scipy/stats/_stats_py.py

```diff
@@ -3147,25 +3147,25 @@ def zmap(scores, compare, axis=0, ddof=0, nan_policy='propagate'):
 
 
 def gstd(a, axis=0, ddof=1):
-    """
+    r"""
     Calculate the geometric standard deviation of an array.
 
     The geometric standard deviation describes the spread of a set of numbers
     where the geometric mean is preferred. It is a multiplicative factor, and
     so a dimensionless quantity.
 
-    It is defined as the exponent of the standard deviation of ``log(a)``.
-    Mathematically the population geometric standard deviation can be
-    evaluated as::
-
-        gstd = exp(std(log(a)))
-
-    .. versionadded:: 1.3.0
+    It is defined as the exponential of the standard deviation of the
+    natural logarithms of the observations.
 
     Parameters
     ----------
     a : array_like
-        An array like object containing the sample data.
+        An array containing finite, strictly positive, real numbers.
```

Comment:
I think it's good that you are more precise here than I was :+1:

Reply:
At some point we'll want to take a pass throughout stats, probably when we change from `array_like` to `array` or whatever (in the array API sense). We very rarely specify.